include/RAJA/policy/atomic_builtin.hpp

```diff
@@ -225,6 +225,19 @@ RAJA_INLINE T builtin_atomic_CAS_oper_sc(T volatile *acc,
 
 }  // namespace detail
 
+template <typename T>
+RAJA_DEVICE_HIP
+RAJA_INLINE void atomicStore(builtin_atomic, T volatile *acc, T value)
+{
+  *acc = value;
+}
+
+template <typename T>
+RAJA_DEVICE_HIP
+RAJA_INLINE T atomicLoad(builtin_atomic, T volatile *acc)
+{
+  return *acc;
```

Comment:
This should also probably use an atomic to read it to ensure atomicity.
I've used something like this before.
`T dummy_for_cas;`
`return AtomicCAS(acc, dummy_for_cas, dummy_for_cas);`

Reply:
I usually prefer `atomicAdd(acc, 0);` for performance, but that or an atomic load would work.